source/includes/day-trips-api.md

```diff
@@ -530,8 +530,9 @@ curl -X POST "https://papi.staging.mydaytrip.net/partners/v1/daytrips/cancel" \
 ```json
 {
     "bookingId": "CB1027",
-    "status": "cancelled",
-    "cancelledAt": "2025-05-18T10:25:00Z"
+    "cancelledAt": "2025-05-18T10:25:00Z",
+    "penalty": 100, // if less than 24hours before departure, otherwise 0
```

Comment:
This comment should be move to the description of the `penalty` parameter. There it will be more noticeable and all related details will be in one place.